ccd-definition/AuthorisationCaseField.json

```diff
@@ -944,6 +972,13 @@
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "children1",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "R"
```

Comment:
Should all other roles only have R?

Reply:
Yes but removing the above block, don't think gatekeeper need read access to children1